### ISSUE TITLE
run 'setup.sh'  failed - CFNetwork SSLHandshake failed (-9806)

### ISSUE THREAD

**Issue by hw20101101** [wireapp/wire-ios#81]

**Comment by hw20101101** [comment]
other question：

`[!] `<PBXBuildFile UUID=`878893641D49113600086646`>` attempted to initialize an object with an unknown UUID. `878893621D490EB600086646` for attribute: `file_ref`. This can be the result of a merge and  the unknown UUID is being discarded.`

**Comment by hw20101101** [comment]
other question：

`ℹ️  Downloading AVS library...
ℹ️  Using wire open source iOS binary
ℹ️  Downloading https://api.github.com/repos/wireapp/avs-ios-binaries/releases/tags/24...
Redirected to https://api.github.com/repos/wireapp/avs-ios-binaries/releases/assets/2025790...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
 17  178M   17 31.1M    0     0  32830      0  1:35:13  0:16:35  1:18:38     0curl: (56) SSLRead() return error -9806`

**Comment by hw20101101** [comment]
other question：


![Uploading QQ20160807-0@2x.png…]()

**Comment by marcoconti83** [comment]
Hi @hw20101101 , thanks for reporting the issue. 
Sadly we can not reproduce this issue, it downloads fine for us. 

From the logs you posted, it seems like you already downloaded a big chunk of the file (31M) before the connection fails. So I'm more inclined to think it's a network error, as it started to transfer and it died only after 31M, rather than a configuration error as the stack overflow answer you posted suggests.

Would it be possible for you to try again from a different network and let us know how it went?

PS: I edited your comment as the link was broken, I fixed it
